operator/docs/user-guides/recover_ingester_pods_in_zone_failure_outages.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "Playbook for recovering ingester pods in zone-failure outages"
+description: "Playbook for recovering ingester pods in zone-failure outages"
+lead: ""
+date: 2023-07-25T08:48:45+00:00
+lastmod: 2023-07-25T08:48:45+00:00
+draft: false
+images: []
+menu:
+  docs:
+    parent: "user-guides"
+weight: 100
+toc: true
+---
+
+This document describes how to recover the ingester pods in the event of a zone failure.
+
+_Disclaimer:_ This document describes a recovery procedure by manually recreating the failed pods in another zone. Right now, we are doing this by deleting PVCs of the impacted pods from the failed zone, so it can be recreated in a different zone. This will cause data loss of the data in the PVC. To avoid actual data loss we always set the replication factor in loki to be 1 or higher so data is always replicated.
+
+## Ingester recovery during a Zone Failure
+
+In a Kubernetes/Openshift cluster, a "zone failure" refers to a situation where nodes or resources in a specific availability zone become unavailable. An availability zone is a distinct location within a cloud provider's data center or region, designed to be isolated from failures in other zones to provide better redundancy and fault tolerance. When a zone failure occurs, it can lead to a loss of services or data if the cluster is not configured properly to handle such scenarios.
+
+This document outlines steps that can be taken to recover ingester pods when there is a zone failure. Ingester pods are deployed as a part of a Statefulset. The Ingester Statefulset also has PVCs associated with the pods. Each Ingester Pod and its associated PVCs are deployed in the same zone as the pod. This is done by using the storageclass.
```

Comment:
I wasn't understanding the last two phrases, feel free to fix it and don't accept this suggestion
```suggestion
This document outlines steps that can be taken to recover ingester pods when there is a zone failure. Ingester pods are deployed as a part of a Statefulset. The Ingester Statefulset also has PVCs associated with the pods. Each Ingester Pod and its associated PVCs are deployed in the same zone.
```

Reply:
I had missed adding the Dynamic provisioning part. Added now. Does it make it more clear now?